src/pr-trigger.ts

```diff
@@ -30,7 +30,7 @@ export async function httpTrigger(context: Context, req: HttpRequest) {
     if (!isDev && !webhooks.verify(req.body, webhooks.sign(req.body))) {
         context.res = {
             status: 500,
-            body: "This webhook did not come from GitHub"
+            body: "This webhook did not come from GitHub",
```

Comment:
These `status`/`body` things remind me of the `tag`/`status`, so I also don't like them.

Reply:
Reverted :heavy_check_mark: